app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
+}
+
+#nav_bar {
+  color: black;
+  background-color: white;
+}
+
+#nav_bar a {
+  color: black;
+}
+
+.home_picture {
+  height: 750px;
+  background: transparent url("homepage.png") no-repeat center -50px;
+  background-size: cover;
+  //margin-left: 25px;
+  //margin-right: 25px;
+
+}
+
+.mdl-layout-title {
+  padding-left: 30%;
+}
+
+.home_navigation {
+  text-align: center;
+  height: 100px;
+}
+
+.plan_trip button {
+  margin-left: 50px;
+  margin-top: 30px;
+  padding: 10px;
+}
+
+.home_about_us {
+  background: yellow;
+}
+
+.home_trip_reviews {
+  background: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.